src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -311,15 +307,15 @@ async Task<JObject> ResolveAsInstanceMember(ArraySegment<string> parts, JObject
                     }
 
                     if (hasCurrentPartNullCondition)
-                        partTrimmed = partTrimmed.Remove(partTrimmed.Length - 1);
-                    JToken objRet = valueOrError.Value.FirstOrDefault(objPropAttr => objPropAttr["name"]?.Value<string>() == partTrimmed);
+                        part = part.Remove(part.Length - 1);
```

Comment:
Shouldn't this be done for both `?`, and `!`?

Reply:
It's done a bit earlier (https://github.com/dotnet/runtime/blob/25e907cf5ecfdec23d37d2d3047ceaf2c14c9a3e/src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs#L269)

Reply:
But we can merge them both (in the place where we currently remove only '?'). I won't change anything